Gems/Terrain/Code/Source/TerrainRenderer/TerrainClipmapManager.h

```diff
@@ -17,6 +17,44 @@
 
 namespace Terrain
 {
+    struct ClipmapConfiguration
```

Comment:
How is the data in this structure getting set? I don't see it exposed on any components and there aren't any CVars.

Reply:
It's currently using the default values on initialization. Because the clipmap system doesn't have a response to editor or config changes at this point. We'll expose this to a data file together with some other settings in the near future.